iOS/MasterTimeline/MasterTimelineViewController.swift

```diff
@@ -156,7 +156,11 @@ class MasterTimelineViewController: UITableViewController, UndoableCommandRunner
 	@objc func showFeedInspector(_ sender: UITapGestureRecognizer) {
 		coordinator.showFeedInspector()
 	}
-	
+
+	@objc func toggleSidebar(_ sender: Any?) {
+		coordinator.toggleSidebar()
+	}
+
```

Comment:
I would have thought that this needed to go into RootSplitViewController to work globally.  Does toggling the sidebar work okay when focus is on the Timeline or Article?

Reply:
Let me double check and get back to you on the behaviour.

Reply:
Double checked. its working when focusing on timeline or article.

Reply:
Moved the `toggleSidebar` to `RootSpiltViewController` and verified it on iPad Pro. Working fine on timeline and article. @vincode-io